src/main/java/aubergine/dance/service/local/LocalFileStorageManager.java

```diff
@@ -0,0 +1,21 @@
+package aubergine.dance.service.local;
+
+import aubergine.dance.service.StorageManager;
+import java.io.File;
+
+public class LocalFileStorageManager implements StorageManager {
+
+    @Override
+    public void createDirectory(String directory) {
+        File file = new File(directory);
+        if (!file.exists()) {
+            file.mkdirs();
+        }
+    }
```

Comment:
how about this?

```java
public void createDirectory(String directory) {
        if (isDirectoryExists(directory)) {
            return;
        }
        File file = new File(directory);
        file.mkdirs();
}
```

pros and cons

pros: method recycle
cons: create file object twice

Reply:
Seems it has better readability, I'll accept that 👍🏻
making file twice looks fine, since it only happens on local profile environment.